### PR TITLE
Staff testing WIP (factoring access token and id token)

### PR DESCRIPTION
**Pull request by dezhizhang1985**

### CODE REVIEW

File: spid_cie_oidc/provider/forms.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+from cProfile import label
```
Comment (peppelinux) [inline spid_cie_oidc/provider/forms.py:1]: Please remove unused imports

Comment (peppelinux) [reply spid_cie_oidc/provider/forms.py:1]:
````
from cProfile import label
from tkinter import Widget
from turtle import width
from attr import attributes
````

File: spid_cie_oidc/provider/templates/op_user_staff_test.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                    <div class="card card-bg no-after">
+                        <div class="card-body pl-lg-0">
+                            <div class="row h-100">
+
+                                <div class="col-12 pl-lg-4">
+
+                                    <div class="mt-2 ml-1">
+                                        <h4 class="text-left">{% trans 'Testing' %}</h4>
+                                        <p class="card-title">
+                                            {% trans "if you reach this page because you are a staff user," %}
```
Comment (peppelinux) [inline spid_cie_oidc/provider/templates/op_user_staff_test.html:21]: please use {% blocktrans %} {% endblocktrans %}

File: spid_cie_oidc/provider/views.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import logging
 import urllib.parse
 import uuid
+import json
```
Comment (peppelinux) [inline spid_cie_oidc/provider/views.py:6]: where the json library is used?

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: spid_cie_oidc/provider/views.py (hunk 3 above)
Comment (dezhizhang1985) [reply spid_cie_oidc/provider/views.py:6]: line 438: "attributes": json.dumps(attributes, indent=4)

**Comment by peppelinux** [comment]
I moved your PR here
https://github.com/italia/spid-cie-oidc-django/pull/118

**Pull request closed without merging**